perceval/components/core_catalog/heralded_cnot.py

```diff
@@ -26,28 +26,25 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
+
 from deprecated import deprecated
-from perceval.components import Circuit, Processor, BS, PERM, Port
+from perceval.components import Circuit, Processor, Source, BS, Port
 from perceval.components.component_catalog import CatalogItem, AsType
+from perceval.components.core_catalog.heralded_cz import HeraldedCzItem
 from perceval.utils import Encoding
 
 
 class HeraldedCnotItem(CatalogItem):
-    article_ref = "https://doi.org/10.1073/pnas.1018839108"
-    description = r"""CNOT gate with 4 heralded modes"""
-    str_repr = r"""                      ╭─────╮
-ctrl (dual rail) ─────┤     ├───── ctrl (dual rail)
-                 ─────┤     ├─────
-                      │     │
-data (dual rail) ─────┤     ├───── data (dual rail)
-                 ─────┤     ├─────
-                      ╰─────╯"""
-    see_also = "postprocessed cnot"
-
-    R1 = 0.228
-    R2 = 0.758
-    theta1 = BS.r_to_theta(R1)
-    theta2 = BS.r_to_theta(R2)
+    article_ref = ""
+    description = r"""Heralded CNOT gate built using Heralded CZ and H"""
```

Comment:
Could you please state that there are only 2 heralded modes in this version of the CNOT gate?
Like:
```python
description = r"""CNOT gate with 2 heralded modes"""
```